dotnet/src/Connectors/Connectors.MistralAI/Client/ContentChunkType.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Connectors.MistralAI.Client;
+
+internal readonly struct ContentChunkType : IEquatable<ContentChunkType>
+{
+    public static ContentChunkType Text { get; } = new("text");
```

Comment:
If you opt in continue with the specific types, suggest moving them to `internal`s

```suggestion
    internal static ContentChunkType Text { get; } = new("text");
```